client/network/src/block_requests.rs

```diff
@@ -336,12 +336,11 @@ where
 	}
 
 	/// Callback, invoked when a new block request has been received from remote.
-	fn on_block_request
-		( &mut self
-		, peer: &PeerId
-		, request: &schema::v1::BlockRequest
-		) -> Result<schema::v1::BlockResponse, Error>
-	{
+	fn on_block_request(
+		&mut self,
+		peer: &PeerId,
+		request: &schema::v1::BlockRequest,
+	) -> Result<schema::v1::BlockResponse, Error> {
```

Comment:
We're removing this in https://github.com/paritytech/substrate/pull/7478
I'd suggest to revert these changes to not cause merge conflicts